app/render/index.ts

```diff
@@ -17,7 +17,10 @@ along with this program; if not, write to the Free Software
 Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 */
 
+require('electron-tabs');
```

Comment:
```suggestion
import 'electron-tabs';
```

Reply:
found this works as well, I would prefer import for code style consistence

Reply:
No problems. I am not native in NodeJS, and I wasn't sure if a specific syntax was required to run the script (in addition to just importing the types from it)